### PR TITLE
chore(test): add @all tag for all tests that are run as part of test:e2e

### PR DESCRIPTION
**Pull request by odockal**
### What does this PR do?
Add @all tag to all e2e tests that should be run on `test:e2e`. And to fix problem with processing grep reg. exp on various platforms by different shells.
### What issues does this PR fix or reference?
#10054 

### How to test this PR?

`pnpm test:e2e` should run all expected tests on all platforms. no failures.

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: tests/playwright/src/specs/deploy-to-kubernetes.spec.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ test.skip(
   'Tests suite should not run on Linux platform',
 );
 
-test.describe.serial('Deploy a container to the Kind cluster', { tag: '@k8s_e2e' }, () => {
+test.describe.serial('Deploy a container to the Kind cluster', { tag: ['@smoke', '@all'] }, () => {
```
Comment (cbr7) [inline tests/playwright/src/specs/deploy-to-kubernetes.spec.ts]: Why did you remove the k8s_e2e tag and added smoke?

File: tests/playwright/src/specs/kind.spec.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ test.afterAll(async ({ runner, page }) => {
   }
 });
 
-test.describe.serial('Kind End-to-End Tests', { tag: '@k8s_e2e' }, () => {
+test.describe.serial('Kind End-to-End Tests', { tag: ['@smoke', '@all'] }, () => {
```
Comment (cbr7) [inline tests/playwright/src/specs/kind.spec.ts]: Same as above.

File: tests/playwright/src/specs/kubernetes-edit-yaml.spec.ts [hunk 3]
```diff
@@ -79,7 +79,7 @@ test.skip(
   'Tests suite should not run on Linux platform',
 );
 
-test.describe.serial('Kubernetes Edit YAML Feature E2E Test', { tag: '@k8s_e2e' }, () => {
+test.describe.serial('Kubernetes Edit YAML Feature E2E Test', { tag: ['@smoke', '@all'] }, () => {
```
Comment (cbr7) [inline tests/playwright/src/specs/kubernetes-edit-yaml.spec.ts]: same as above.

File: tests/playwright/src/specs/kubernetes.spec.ts [hunk 4]
```diff
@@ -87,7 +87,7 @@ test.skip(
   'Tests suite should not run on Linux platform',
 );
 
-test.describe('Kubernetes resources End-to-End test', { tag: '@k8s_e2e' }, () => {
+test.describe('Kubernetes resources End-to-End test', { tag: ['@smoke', '@all'] }, () => {
```
Comment (cbr7) [inline tests/playwright/src/specs/kubernetes.spec.ts]: same as above.

**Review by cbr7 — CHANGES_REQUESTED** [review]
I don't really understand why we need an @all tag at all, if you want to run all the tests you can just select the folder where all the test files are located, that will run all the tests. Adding a tag to all test files means that you don't need to add it to any test file.

**Comment by odockal** [comment]
@cbr7 Because:
> Running E2E tests on windows using npm target test:e2e is limited by an os configuration, this command in particular: `npx playwright test tests/playwright/src/specs/ --grep-invert '@update-install|@podman-remote'"`. On linux and mac all works ok, but on windows the reg exp expression is processed with cmd or powershell and simply tries to run podman-remote as a command and fails on error.

I would love to keep it as it is, but I simply haven't found a way to make it work universally on all platforms. I can try a couple more reg exps and escaping if it fill all, but nothing worked for me me so far.

**Comment by cbr7** [comment]
@odockal Wouldn't it be just simpler to move the update and podman-remote e2e tests into another folder? There is no particular reason they need to be under `/specs/`, you can move them to `tests/playwright/src/special_tests/<custom folder for each test suite here>` and that will solve all problems permanently, since you will call always whichever folder you want to run, without any params at all.

**Comment by odockal** [comment]
> @odockal Wouldn't it be just simpler to move the update and podman-remote e2e tests into another folder? There is no particular reason they need to be under `/specs/`, you can move them to `tests/playwright/src/special_tests/<custom folder for each test suite here>` and that will solve all problems permanently, since you will call always whichever folder you want to run, without any params at all.

that might work!

**Comment by odockal** [comment]
Closing as better and simpler solution proposed.

**Comment by odockal** [comment]
New PR opened to have clear review: https://github.com/podman-desktop/podman-desktop/pull/10060
